emf/multipkg.json

```diff
@@ -10,6 +10,15 @@
     "eSuperTypes" : [ {
       "eClass" : "http://www.eclipse.org/emf/2002/Ecore#//EClass",
       "$ref" : "https://strumenta.com/kolasu/v1#//ASTNode"
+    } ],
+    "eStructuralFeatures" : [ {
```

Comment:
This I think would not be necessary because this information is already represented in EMF as the EClass of the node